app/routes/explore.js

```diff
@@ -1,30 +1,29 @@
 import Ember from 'ember';
-import moment from 'moment';
 
 const { Route, RSVP } = Ember;
 
 export default Route.extend({
   titleToken() {
     return this.l10n.t('Explore');
   },
+  queryParams: {
+    category: {
+      refreshModel: true
+    },
+    subCategory: {
+      refreshModel: true
+    },
+    eventType: {
+      refreshModel: true
+    },
+    dateRange: {
+      refreshModel: true
+    }
+  },
 
-  model() {
+  model(params) {
     return RSVP.hash({
-      events: this.store.query('event', {
-        sort   : 'starts-at',
-        filter : [
-          {
-            name : 'starts-at',
-            op   : 'ge',
-            val  : moment().toISOString()
-          },
-          {
-            name : 'state',
-            op   : 'eq',
-            val  : 'published'
-          }
-        ]
-      }),
+      events      : this.store.query('event', params),
```

Comment:
How can you pass in the params directly like this ? That is not what the server expects.